cmd/service/test.go

```diff
@@ -38,6 +38,7 @@ func init() {
 	Test.Flags().StringP("event-filter", "e", "*", "Only log the data of the given event")
 	Test.Flags().StringP("task-filter", "r", "", "Only log the result of the given task")
 	Test.Flags().StringP("output-filter", "o", "", "Only log the data of the given output of a task result. If set, you also need to set the task in --task-filter")
+	Test.Flags().BoolP("full-logs", "f", false, "Display logs from service and its dependencies")
```

Comment:
The flag `full-logs` is not explicit enough.
I suggest the name `logs-dependencies` and as explanation: "Also displays the logs of all service's dependencies".
Also please add this flag in the Example of this command.

Reply:
`logs-dependencies` is even more confusing I think because we don't know if we will have only the dependencies or all

Reply:
`logs-all` ?